### ISSUE TITLE
Is it possible to use non standard shape libraries? 

### ISSUE THREAD

**Issue by stoney95** [LukeCarrier/mkdocs-drawio-exporter#44]
Using the exporter works fine. But the exports don't display Icons / Shapes that are coming from non-standard shape libraries. 

When drawing my diagram I added them with "+ More Shapes" and then selecting - in my case - the Azure shapes.

<img width="389" alt="Screenshot 2023-02-01 at 13 17 42" src="https://user-images.githubusercontent.com/34100018/216040409-a27abd01-962b-4abe-97a3-2f59b0828939.png">

**Comment by LukeCarrier** [comment]
Not right now, but this is a great enhancement request 🙂

I can't commit to ETAs right now as I'm not actively using this plugin but I will keep the issue open in case I or someone else has a go at implementing it.

**Comment by codymwalker** [comment]
I saw the same behavior with non-standard shapes. It looks like this is possible without any code changes though.
Testing locally with the draw.io CLI arguments, I was able to get the shapes showing by passing `--embed-svg-images`.
You have the ability to pass additional arguments to the exporter via the MkDocs config, which get added to the underlying command.

```
plugins:
    - drawio-exporter:
        format: svg
        drawio_args: [--embed-svg-images]
```

Hope that helps @stoney95.
